qemu/tests/kvm_stat.py

```diff
@@ -0,0 +1,82 @@
+import logging
+import os
+import subprocess
+
+from avocado.utils import process
+from virttest import env_process
+from virttest import error_context
+from virttest import utils_misc
+from virttest import utils_package
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    1.install kernel-tools package in host
+    2.modify kvm_stat.service file and start kvm_stat.service
+    3.run VM
+    4.use "logrotate" to separate the log file
+    5.Check the generated log file
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment
+    """
+    interval = float(params.get("interval_time", "10"))
+
+    def tweak_file(filename, old_str, new_str):
+        # replace the strings in the file
+        file_data = ""
+        with open(filename, "r") as f:
+            for line in f:
+                if old_str in line:
```

Comment:
What's the expected result if no `old_str` in the file content ?  Currently, the new value will not be set and test will continue, i think it is not suitable.